charts/portworx/templates/portworx-controller.yaml

```diff
@@ -91,7 +91,7 @@ spec:
       imagePullSecrets:
         - name: {{ required "A registry secret is required for openshift installation" .Values.registrySecret }}
      {{- else }}
-      {{- if not empty .Values.registrySecret }}
+      {{- if not (empty .Values.registrySecret) }}
```

Comment:
@orbiran88 thank for the PR.

Can you have sample outputs of the failure before and after the fix?

Overall the change looks good.

Reply:
ping @orbiran88 ^^